models/public/colorization-v2-norebal/colorization-v2-norebal.md

```diff
@@ -25,12 +25,13 @@ Model give as output predict A- and B-channels of LAB-image.
 
 ## Accuracy
 
-The quality metrics calculated on ImageNet validation dataset.
+The accuracy metrics calculated on ImageNet validation dataset.
+The accuracy metrics used VGG-16 classification model.
```

Comment:
it is very simple language and unclear how it is connected with colourization (from the point of view of a person who is not familiar with use case), I think need more scientific. Something like:
`The accuracy metrics calculated on ImageNet validation dataset using VGG16 model and colourization as preprocessing.`

Reply:
also it makes sense to have hyper link on vgg16 description instead model name or specify the framework (models from different frameworks can give different results, if in future we will have vgg16 model from another framework we need correctly preserve experiments conditions for reproducing)

Reply:
And I think, you should explain, what metric value means. If it mean result of VGG validation with rgb->gray->coloriaztion preprocessing, you should provide original value too, for comparison.

Reply:
@eizamaliev @eaidova done)

Reply:
it is not related to description, but do you not think that the difference from results can be connected with unexpected color space?
I mean VGG16 expects BGR images,  After colorization, do you convert images to bgr for inference or not?

Reply:
@eaidova colorization return BGR image in AC, and accuracy same as article metrics

Reply:
ok)